server/src/com/cloud/network/security/SecurityGroupListener.java

```diff
@@ -81,7 +81,7 @@ public boolean isRecurring() {
     @Override
     public boolean processAnswers(long agentId, long seq, Answer[] answers) {
         List<Long> affectedVms = new ArrayList<Long>();
-        int commandNum = 0;
+        // int commandNum = 0;
```

Comment:
While making changes in this file, I noticed that Eclipse was pointing out that the variable commandNum was only written to (never read from). Instead of removing the entire line, I just commented it out in case someone later wanted to easily resume making use of this variable.